docs/contributing/how-to-contribute.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: How to Contribute
+शीर्षक: योगदान कैसे करें
```

Comment:
`शीर्षक` => `title`

we should not translate this. This will be used while rendering on webpage